skipchain/api.go

```diff
@@ -23,6 +23,9 @@ const (
 	ErrorVerification
 	// ErrorOnet indicates an error from the onet framework
 	ErrorOnet
+	// ErrorBlockInProgress indicates that currently a block is being formed
+	// and propagated
+	ErrorBlockInProgress
```

Comment:
Not only that, but also if a block is begin formed and you want to add a block at the same index as the one being formed.